### PR TITLE
feat: Cleanup Review's importer | NPG-7782

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

Cleanup review's importer logic, updated configuration process, load config from file

### CODE REVIEW

**Review by saibatizoku — APPROVED** [review]
LGTM

File: utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{  
+    "event_id": 0,
+    "group_id": 37429,
+    "review_stage_ids": [171],
+    "nr_allocations": [1, 1],
```
Comment (minikin) [inline utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json:5]: I'm having trouble understanding the meaning of `nr_allocations`. Can you clarify what `nr` refers to?

File: utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "campaign_group_id": 88,
+    "questions": {
+        "Question 1": "Impact / Alignment",
+        "Question 2": "Feasibility",
+        "Question 3": "Auditability"
+    },
+    "stage_ids": [4684, 4685, 4686],
+    "anonymize_start_id": 5000
```
Comment (minikin) [inline utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json:13]: I believe that we can come up with a better name for the field "anonymize_start_id".
Could you clarify the purpose of `anonymize_start_id`? I'm not entirely clear on what it does.

**Review by minikin — CHANGES_REQUESTED** [review]

File: utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json (hunk 1 above)
Comment (Mr-Leshiy) [reply utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json:5]: nr - number of allocations. I will rename it

File: utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json (hunk 2 above)
Comment (Mr-Leshiy) [reply utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json:13]: It is id to start with to assign anonymized IDs

**Review by minikin — APPROVED** [review]

**Pull request merged**